### DISCUSSION TITLE
Installation of Monailabel

### DISCUSSION THREAD

**Discussion by D123Roy** [Project-MONAI/MONAI#5266] [Q&A]
Installation of monailabel stops at installing the dependencies for gdown. The error message is attached. (pip subprocess to install build dependencies did not run successfully.)
`Collecting gdown==4.5.1
  Using cached gdown-4.5.1.tar.gz (14 kB)
  Installing build dependencies ... error
  error: subprocess-exited-with-error`

  `× pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]`
      `WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError`
`× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.`

`note: This error originates from a subprocess, and is likely not a problem with pip.`
Please note - I am using an updated pip, with Python version -3.8.10. I tried to install the package separately `pip --proxy <proxy-name> gdown` with the same resulting error.

**Comment by wyli** [comment]
since the error is SSLError(SSLCertVerificationError, have you tried for example `pip install gdown  --global http.sslVerify false` (according to https://stackoverflow.com/a/29751768). cc @SachidanandAlle

**Reply by D123Roy** [reply]
I am so grateful for your response. The issue resolved !
